### ISSUE TITLE
sched packaging version not updating correctly

### ISSUE THREAD

**Issue by trws** [flux-framework/flux-core#1375]
The module file emmitted from a packaged flux-sched seems to be pinned at an old version, loading 0.3.0 gives a path to 0.1.0 (or similar, this may be off a digit).

**Comment by grondo** [comment]
I see the problem (though this is TOSS2.0 specific, we don't use environment-modules for flux on TOSS3.x)

I'm not even sure if we were building new packages for TOSS2, but it is ok if the "fix" goes in with the next update of flux-sched (0.4.0 I'd assume?)

If not, I could build a new version of 0.3.0, but I have no idea when it would actually go out to TOSS2 systems.

**Comment by trws** [comment]
Yeah, I just wanted to document it since I hit it.  Mainly for if we 
start having multiple version around for toss3 etc.
